lib/ts/recipe/emailverification/emailVerificationClaim.ts

```diff
@@ -28,6 +28,13 @@ export class EmailVerificationClaimClass extends BooleanClaim {
                 refresh: this.refresh,
                 shouldRefresh: (payload, userContext) => {
                     const DateProvider = DateProviderReference.getReferenceOrThrow().dateProvider;
+                    const currentTime = DateProvider.now();
+
+                    if (this.shouldRefreshLastCalled && this.shouldRefreshLastCalled > currentTime - 1000) {
```

Comment:
Please make this an explicit check for checking if `shouldRefreshLastCalled` is not undefined.